contracts/lib/Organization.sol

```diff
@@ -83,16 +117,40 @@ contract Organization is OrganizationInterface, IsMemberInterface {
     /* Constructor */
 
     /**
-     * @notice The address that creates the contract is set as the initial
-     *         owner.
+     * @notice 
+     *
+     * @param _owner The address that shall be registered as the owner of the
+     *               organization.
+     * @param _admin The address that shall be registered as the admin of the
+     *               organization. Can be address(0) if no admin is desired.
+     * @param _workers An array of initial worker addresses. Can be an empty
+     *                 array if no workers are desired or known at construction.
+     * @param _expirationHeight If any workers are given, this will be the
+     *                          block height at which they expire.
      */
-    constructor() public
+    constructor(
+        address _owner,
+        address _admin,
+        address[] memory _workers,
+        uint256 _expirationHeight
+    )
+        public
     {
-        owner = msg.sender;
+        require(
+            _owner != address(0),
+            "The owner must not be the zero address."
+        );
+
+        owner = _owner;
+        admin = _admin;
+
+        for(uint256 i = 0; i < _workers.length; i++) {
+            setWorkerInternal(_workers[i], _expirationHeight);
+        }
     }
 
 
-    /* External Functions */
+    /* External functions */
```

Comment:
Functions ;)

Reply:
We actually have more `External functions` in the code base than `External Functions` ;)